>From c870912992da04d5513e75ea40b2b20f69184333 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Sun, 4 Jan 2015 12:56:57 +0900
Subject: [PATCH 2/2] add debian/control file for Emacs add-on

Emacs package must have "Depends: emacsen-common (>= 2.0.8)",
see debian-emacs-policy

>5) Emacs add-on package support (examples below should make this much clearer)
>
>  A) Each add-on package must add a "Depends: emacsen-common (>=
>     2.0.8)" and include a file like this that indicates its
>     emacsen-common compatibility level:
---
 lib/emacs/control | 15 +++++++++++++++
 1 file changed, 15 insertions(+)
 create mode 100644 lib/emacs/control

diff --git a/lib/emacs/control b/lib/emacs/control
new file mode 100644
index 0000000..d850f09
--- /dev/null
+++ b/lib/emacs/control
@@ -0,0 +1,15 @@
+Source: #PACKAGE#
+Section: lisp
+Priority: optional
+Maintainer: #USERNAME# <#EMAIL#>
+Build-Depends: #BUILD_DEPS#
+Standards-Version: #POLICY#
+Homepage: <insert the upstream URL, if relevant>
+#Vcs-Git: git://anonscm.debian.org/collab-maint/#PACKAGE#.git
+#Vcs-Browser: http://anonscm.debian.org/cgit/collab-maint/#PACKAGE#.git/
+
+Package: #PACKAGE#
+Architecture: all
+Depends: ${misc:Depends}, emacsen-common (>= 2.0.8), emacs | emacsen,
+Description: <insert up to 60 chars description>
+ <insert long description, indented with spaces>
-- 
2.1.4

